ui/public/iframe-test.html

```diff
@@ -9,9 +9,9 @@
       function changeApp(app, lang) {
```

Comment:
This file will be packed and embedded into PD. Please remove it.

Reply:
Got it, will remove it later.

Reply:
Done